scheduler/plugin/plugin.go

```diff
@@ -196,16 +196,37 @@ func (pl *wasmPlugin) AddPod(ctx context.Context, state *framework.CycleState, p
 	if pl.guestInterfaces&iPreFilterExtensions == 0 {
 		return nil // unimplemented
 	}
-	panic("TODO: scheduling: AddPod")
+
+	// Add the stack to the go context so that the corresponding host function
+	// can look them up.
+	params := &stack{pod: podToSchedule}
+	ctx = context.WithValue(ctx, stackKey{}, params)
+	status := &framework.Status{}
```

Comment:
Please change the line 194 like below.


`func (pl *wasmPlugin) AddPod(ctx context.Context, state *framework.CycleState, podToSchedule *v1.Pod, podInfoToAdd *framework.PodInfo, nodeInfo *framework.NodeInfo) (status *framework.Status) {
`